sdk/identity/identity/integration/AzureWebApps/package.json

```diff
@@ -12,7 +12,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/identity": "^4.0.0",
+    "@azure/identity": "dev",
```

Comment:
I acknowledge that we should ideally build and pack the source code instead of using the nightly dev version. This may be a good enough compromise (recall that folks may not be running integration tests on PRs anyway) but I might do a follow-up that builds and packs the source code for all the integration tests

Reply:
yes, but this test package is not part of rush. Rush enables us to build it against the source code. This one will be tricky. Any thoughts on how we can do?

Reply:
Yes, I have thoughts! 

What I was thinking at a high level is to run `rushx pack` in the `identity/identity` directory.

That creates a file like `azure-identity-4.1.0-beta.2.tgz` which we can then copy to the test directories

In AKS that would be copying it into the docker container and then running `npm install` inside the container
For other files, not 100% sure

As for the details 🤷 

Using `dev` is honestly close enough that it may not be worth the effort, but maybe during some downtime it would be a good thing to try

Reply:
Yes though we may have to add those steps in the pipeline itself